params.py

```diff
@@ -117,7 +122,7 @@
     # 'current_set_up': ['openface', 'facenet', 'fbdeepface', 'vggface',
     #                     'arcface','lresnet', 'ir50m1sm', 'ir50asia',
     #                     'face_recognition'],
-    'current_set_up': ['lresnet'],
+    'current_set_up': ['face_recognition'],
```

Comment:
`RuntimeError: Unsupported image type, must be 8bit gray or RGB image.` 
Deze ligt denk ik meer aan de branch van Andrea, want met vggface doet hij het wel. Dus kan het voor deze branch ook zo laten.

Reply:
k